app/views/admin/registrations/_questions.html.haml

```diff
@@ -1,9 +1,5 @@
-- @conference.questions.each do |q|
-
-  %b Q:
-  = q.title
-
-  %b A:
-  - registration.qanswers.where(question_id: q.id).each do |qa|
-    = qa.answer.title
-  %br
+- cache [:admin, @conference, registration.qanswers] do
+  %dl
```

Comment:
why are you changing `%b` by `%dl`? how is that related to the performance? :confused:

Reply:
The Question partial had to be redesigned to accommodate the different data model.